src/app/eslint/index.js

```diff
@@ -30,6 +31,12 @@ async function eslint() {
     (eslintChoice) => eslintChoice.name === lintChoice
   )
 
+  const { hookChoice } = await askHookChoices()
+
+  if (hookChoice) {
+    githook()
+  }
+
```

Comment:
Would it make sense to ask this later in the process, after the linter is already configured? Which makes me realize that the `githook` command is only adapted to hook ESLint, what about Stylelint?

Reply:
I tested it after adding the eslint files but it feels a bit odd to select, wait a few seconds until all is set up and as again instead of just asking everything at the begining and then adding what needs to be added

Reply:
Fair enough